modules/ServerAPI/src/server/CustomerSpecificServerFunctions.coffee

```diff
@@ -234,15 +244,25 @@ exports.ssoLoginStrategy = (req, profile, callback) ->
 
 	if config.all.server.security.saml.roles.sync
 		console.log "Checking for roles to sync"
-		# Check the users ldap roles against the saved roles
+
+		# Get the groups from the profile and put them into ACAS format [{roleName: , lsType: 'System', lsKind: }]
 		ssoSystemRoles = getSystemRolesFromSSOProfile(profile)
+		console.log("Found #{ssoSystemRoles.length} 'ACAS'/'CRREG' roles in sso profile for author #{savedAuthor.userName}, #{JSON.stringify(ssoSystemRoles)}")
```

Comment:
nit: We actually just check for `CMPDREG` and then assume any groups aside from that are ACAS-related even if they don't have `'ACAS'`. At minimum I would correct `CRREG` to `CMPDREG` in this logging line.

Reply:
I went with the minimum here. I added notes to the function to explain what we are doing.